Refrigerator/Program.cs

```diff
@@ -0,0 +1,223 @@
+                Console.WriteLine("100: Exit");
+
+                int choice = int.Parse(Console.ReadLine());
+
+                switch (choice)
+                {
+                    case 1:
+                        Console.WriteLine(refrigerator1.ToString());
+                        break;
+                    case 2:
+                        Console.WriteLine($"There is {refrigerator1.GetAvailableSpace()} square meters of free space left in the refrigerator.");
+                        break;
+                    case 3:
+                        Item itemToAdd = GetItem();
+
+                        if (refrigerator1.AddItem(itemToAdd))
+                            Console.WriteLine("item Added.");
+                        else Console.WriteLine("There is no available space for this item.");
+                        break;
+                    case 4:
+                        Console.WriteLine("Enter item Id that you want to take out of the refrigerator: ");
+                        int itemId = int.Parse(Console.ReadLine());
+                        if (refrigerator1.RemoveItem(itemId) != null)
+                            Console.WriteLine("item removed.");
+                        else Console.WriteLine("item Id does not exist.");
+                        break;
+                    case 5:
+                        List<Item> expiredItems = new List<Item>();
+                        expiredItems = refrigerator1.CleanRefrigerator();
+                        if (expiredItems.Count > 0)
+                        {
+                            Console.WriteLine("item removed:");
+                            foreach (Item item in expiredItems)
+                                Console.WriteLine(item.ToString());
+
+                        }
+                        else Console.WriteLine("There are no expired items.");
+                        break;
+                    case 6:
+                        Console.WriteLine("What would you like to get:");
+                        ItemType itemType = GetItemType();
+                        Kashrut kashrut = GetKashrut();
+
+                        List<Item> userFoodChoices = refrigerator1.FindFood(kashrut, itemType);
+                        if (userFoodChoices.Count > 0)
+                        {
+                            Console.WriteLine("The items that suit your choice: ");
+                            foreach (var item in userFoodChoices)
+                                Console.WriteLine(item.ToString());
+                        }
+                        else Console.WriteLine("There are no items that suit your choice.");
+                        break;
+                    case 7:
+                        List<Item> items = refrigerator1.SortItemsByExpiryDate();
+                        if (items.Count >0)
+                        {
+                            Console.WriteLine("The items in the refrigerator are listed by expiration date:");
```

Comment:
build a class of logger to centralized your logic of prints.

class logger would help